website_docs/manual.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Manual Instrumentation
 linkTitle: Manual
-aliases: [/docs/instrumentation/ruby/manual_instrumentation]
+aliases: [/docs/instrumentation/ruby/manual_instrumentatio, /docs/instrumentation/ruby/events, /docs/instrumentation/ruby/context-propagation]
```

Comment:
`manual_instrumentatio` -> `manual_instrumentation` ?

Reply:
of course, thanks!

Reply:
```suggestion
aliases: [/docs/instrumentation/ruby/manual_instrumentation, /docs/instrumentation/ruby/events, /docs/instrumentation/ruby/context-propagation]
```